src/app/ApplicationTemplate.Shared.Views/Content/Authentication/CreateAccountPage.xaml

```diff
@@ -129,18 +129,19 @@
 					<!-- Birth Day Label -->
 					<TextBlock Text="Date of birth"
 							   x:Uid="CreateAccount_DateOfBirth"
-							   Style="{StaticResource Subtitle2}"
-							   Foreground="{StaticResource MaterialOnPrimaryBrush}"
+							   Style="{StaticResource TitleSmall}"
+							   Foreground="{StaticResource OnPrimaryBrush}"
 							   Margin="0,12,0,0" />
 
 					<!-- Birth Day DatePicker -->
 					<local:DataValidationView PropertyName="DateOfBirth"
 											  Model="{Binding Form}">
-						
+						<!-- On Uno Material V2 the Background and the Foreground properties don't take the correct values -->
+						<!-- Issue: https://github.com/unoplatform/Uno.Themes/issues/969 -->
 						<CalendarDatePicker x:Name="BirthDayField"
 											Date="{Binding Form.DateOfBirth, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
-											Background="{StaticResource MaterialSurfaceBrush}"
-											Foreground="{StaticResource MaterialOnSurfaceBrush}"
+											Background="{StaticResource SurfaceBrush}"
+											Foreground="{StaticResource OnSurfaceBrush}"
```

Comment:
Looking at your gifs, it seems that something might not work as expected ?

![image](https://user-images.githubusercontent.com/15191066/230225484-61e245de-5a99-45f4-8e6c-f514112d84fc.png)

Reply:
yes, i've created an issue for this
https://github.com/unoplatform/Uno.Themes/issues/969